src/main/g8/project/plugins.sbt

```diff
@@ -1,4 +1,4 @@
 // The Lagom plugin
-addSbtPlugin("com.lightbend.lagom" % "lagom-sbt-plugin" % "$lagom_version$")
+addSbtPlugin("com.lightbend.lagom" % "lagom-sbt-plugin" % "1.4.0")
 // Needed for importing the project into Eclipse
-addSbtPlugin("com.typesafe.sbteclipse" % "sbteclipse-plugin" % "5.1.0")
+addSbtPlugin("com.typesafe.sbteclipse" % "sbteclipse-plugin" % "5.2.4")
```

Comment:
Is the `sbteclipse` update required? If so, we should add to the migration guide.

Reply:
Just saw your comment in https://github.com/lagom/lagom-scala.g8/pull/24

I think we'll do a separate sbt migration guide after 1.4, so we can hold off until then.